c++ training/W2S1/E3/Source.cpp

```diff
@@ -0,0 +1,93 @@
+#include <iostream>
+
+//Exercise 3 - Shapes and Areas
+//Create two classes : a Circle class and a Square class.For each class, take in the necessary information to calculate its areaand perimeter.Remember to validate input.
+//
+//Ask the user if they want to create a circle or a square.For their pick, ask the necessary information to create the shape.Then, tell them the areaand the perimeter of their shape.
+//
+//Your class should have the following methods :
+//ExampleShape();
+//ExampleShape(/* Params */);
+//float GetArea();
+//float GetPerimeter();
+
+class Circle {
+private:
+	float m_radius;
+public:
+	Circle() {
+		m_radius = 1.0f;
+	}
+	Circle(float radius) {
+		m_radius = radius;
+	}
+	float GetArea() {
+		return 3.14159265358979323846f *m_radius * m_radius;
+	}
+	float GetPerimeter() {
+		return 2 * 3.14159265358979323846f * m_radius;
```

Comment:
Nice use of the f for the PI float, but you're using 2 as an integer and multiplying it by a float. Why not make it a float from the beginning?